Content.Client/UserInterface/GameHud.cs

```diff
@@ -248,24 +248,24 @@ public void Initialize()
 
             _buttonSandboxMenu.OnToggled += args => SandboxButtonToggled?.Invoke(args.Pressed);
 
-            // Tutorial
-            _buttonTutorial = new TopButton(tutorialTexture, ContentKeyFunctions.OpenTutorial, _inputManager)
+            // Info Window
+            _buttonInfo = new TopButton(infoTexture, ContentKeyFunctions.OpenInfo, _inputManager)
             {
-                ToolTip = Loc.GetString("Open tutorial."),
+                ToolTip = Loc.GetString("Open server info."),
```

Comment:
are you sure you changed the corresponding loc.getstring

Reply:
I wasn't going to change it here since someone else can go through GameHud.cs and change them all at once. I'll push it I guess.